crates/sui-indexer/src/apis/indexer_api.rs

```diff
@@ -275,6 +269,33 @@ where
             has_next_page,
         })
     }
+}
+
+#[async_trait]
+impl<S> IndexerApiServer for IndexerApi<S>
+where
+    S: IndexerStore + Sync + Send + 'static,
+{
+    async fn get_owned_objects(
+        &self,
+        address: SuiAddress,
+        query: Option<SuiObjectResponseQuery>,
+        cursor: Option<CheckpointedObjectID>,
+        limit: Option<usize>,
+    ) -> RpcResult<ObjectsPage> {
+        if !self
+            .migrated_methods
+            .contains(&"get_owned_objects".to_string())
+        {
+            return self
+                .fullnode
+                .get_owned_objects(address, query, cursor, limit)
```

Comment:
This change is fine, but is get_owned_objects existing on FN right now? Or did it get moved to suix api only.

Reply:
it's here: https://github.com/MystenLabs/sui/blob/main/crates/sui-json-rpc/src/indexer_api.rs#L67

Reply:
https://github.com/MystenLabs/sui/blob/04f29b4170b9a52a73e5a2b9639c2b593b2c308c/crates/sui-json-rpc/src/indexer_api.rs#L67